ToDoApp/ToDoApp/ToDoService.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ToDoDetail.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-11-29.
+//
+
+import Foundation
+
+enum TaskError: Error {
+    case unavailableTask
+}
+
+struct Task {
+    var id: UUID = UUID()
```

Comment:
id should be a constant since we will never change it

Reply:
This has been fixed.